🐛 Nightly builds of @biomejs/cli-* don't publish CLI binary

It would appear that none of the published NPM packages for the v1.8.4 nightly builds of `@biomejs/cli-linux-x64` include the `biome` binary, which I imagine is probably the only reason this package is published at all.

- [@biomejs/cli-linux-x64@v1.8.4-nightly.d079e53 files](https://www.npmjs.com/package/@biomejs/cli-linux-x64/v/1.8.4-nightly.d079e53?activeTab=code)
- [@biomejs/cli-linux-x64@1.8.4-nightly.bd1d0c6 files](https://www.npmjs.com/package/@biomejs/cli-linux-x64/v/1.8.4-nightly.bd1d0c6?activeTab=code)
- [@biomejs/cli-linux-x64@1.8.4-nightly.a579bf7 files](https://www.npmjs.com/package/@biomejs/cli-linux-x64/v/1.8.4-nightly.a579bf7?activeTab=code)
- [@biomejs/cli-linux-x64@1.8.3 files](https://www.npmjs.com/package/@biomejs/cli-linux-x64/v/1.8.3?activeTab=code)
- [@biomejs/cli-linux-x64@1.7.4-nightly.125f34b files](https://www.npmjs.com/package/@biomejs/cli-linux-x64/v/1.7.4-nightly.125f34b?activeTab=code)

### What happened?

Run the following on any x86 linux host:

```
npx @biomejs/biome@nightly
```

This results in the following error, or similar: `Error: Cannot find module '@biomejs/cli-linux-x64/biome'`

### Expected result

It should work as normal.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

--- a/packages/@biomejs/biome/scripts/generate-packages.mjs
+++ b/packages/@biomejs/biome/scripts/generate-packages.mjs
@@ -58,11 +58,14 @@ function copyBinaryToNativePackage(platform, arch) {
 	);
 	const binaryTarget = resolve(packageRoot, `biome${ext}`);
 
-	if (fs.existsSync(binaryTarget)) {
-		console.log(`Copy binary ${binaryTarget}`);
-		fs.copyFileSync(binarySource, binaryTarget);
-		fs.chmodSync(binaryTarget, 0o755);
+	if (!fs.existsSync(binarySource)) {
+		console.error(`Source for binary for ${buildName} not found at: ${binarySource}`);
+		process.exit(1);
 	}
+
+	console.log(`Copy binary ${binaryTarget}`);
+	fs.copyFileSync(binarySource, binaryTarget);
+	fs.chmodSync(binaryTarget, 0o755);
 }
 
 function updateWasmPackage(target) {

